lib/lure/vcs/hg.go

```diff
@@ -1,11 +1,14 @@
-package lure
+package vcs
 
 import (
 	"errors"
 	"fmt"
 	"os"
 	"regexp"
 	"strings"
+
+	"github.com/coveooss/lure/lib/lure/log"
+	osUtils "github.com/coveooss/lure/lib/lure/os"
```

Comment:
We have an alias here, but "Git.go" has no alias. It would be best to either use `osUtils` or `os` in both places, just pick your preference :)